app/services/workflow_service/property_assessment.rb

```diff
@@ -0,0 +1,68 @@
+module WorkflowService
+  class PropertyAssessment < BaseWorkflowService
+    def call
+      calculate_property
+      true
+    end
+
+    private
+
+    def calculate_property
+      @remaining_mortgage_allowance = Threshold.value_for(:property_maximum_mortgage_allowance, at: @submission_date)
+      calculate_additional_properties
+      calculate_main_dwelling(request_data: @particulars.request.applicant_capital.property.main_home,
+                              response_data: @particulars.response.details.capital.property.main_dwelling)
+    end
+
+    def calculate_additional_properties
+      @particulars.request.applicant_capital.property.additional_properties.each do |additional_property|
+        response_data = OpenStruct.new(AssessmentParticulars.initial_property_details)
+        calculate_individual_property(request_data: additional_property, response_data: response_data, property_type: :additional_property)
```

Comment:
I'm not a fan of these `request_data` and `response_data` names...
The `_data` part is not useful and can just be dropped.
Are there better names we can use? Especially for `request_data`:
`property`, `dwelling`, etc... ?

Reply:
I've replaced them with `property:` and `assessment:`